dist/react-onclickoutside.js

```diff
@@ -293,7 +292,9 @@ function onClickOutsideHOC(WrappedComponent, config) {
         }
       }
 
-      this.componentNode = reactDom.findDOMNode(this.getInstance());
+      this.componentNode = reactDom.findDOMNode(this.getInstance()); // return early so we dont initiate onClickOutside
+
+      if (this.props.disableOnClickOutside) return;
```

Comment:
onclickoutside will never initialize if disableOnClickOutside will update (from true to false or from false to true)

Reply:
@Pomax

Reply:
Im trying to understand what you are saying, but isnt that the point of passing disable so it does not get initialised. and if you add it to componentDidUpdate, the handleclickoutside will be initialised.

Reply:
If you pass disabled as true and then you change props disabled to false (in component in your app), outside will not be initialized, so will not work..

Reply:
a quick solution would be to return early in your handclickoutside if your prop is disabled. reverting this commit will make it so its never disabled.